docs/remix_as_code_viewer.md

```diff
@@ -0,0 +1,38 @@
+Remix as code viewer
+=============================
+
+Through Etherscan
+------------------
+
+If there is a contract verified on Etherscan and someone wants to have a look on it in an IDE, mostly for a mutiple part contract verification, Remix provides a quick way to load the contract.
+
+Usually, contract is shown on Etherscan on a URL like this:
```

Comment:
This sentence could be:
A typical Etherscan URL looks like this:
or
Usually, a contract is shown on Etherscan with a URL like this:

Reply:
updated